readthedocs/templates/builds/build_detail.html

```diff
@@ -55,10 +55,50 @@
       {% endif %}
     </ul>
 
-    <div class="build-id">
-      {% blocktrans with build_id=build.pk %}
-        Build #{{ build_id }}
-      {% endblocktrans %}
+    <div class="build-title">
+
+      <span class="build-id">
+        <span class="build-state-normal"
+              data-bind="visible: !finished()">
+          {% blocktrans with build_id=build.pk %}
+            Build #{{ build_id }}
+          {% endblocktrans %}
+        </span>
+
+        <span class="build-state-successful"
+              data-bind="visible: finished() && success()"
+              style="display: none;">
+          {% blocktrans with build_id=build.pk %}
+            Build #{{ build_id }}
+          {% endblocktrans %}
+        </span>
+
```

Comment:
I don't know how to do this on a better way, what js library are you using?

Reply:
Knockout I believe.